packages/expo-video/android/src/main/res/layout/fullscreen_player_activity.xml

```diff
@@ -11,6 +11,5 @@
   <androidx.media3.ui.PlayerView
     android:id="@+id/player_view"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" />
```

Comment:
You deleted a bit too much 😅

Reply:
woops, sorry about that @behenate  🙈🙈🙈 fixed now